### PR TITLE
AC Test Fix - LRAC-10807 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10807
[Testray](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1658148428)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 1]
```diff
@@ -95,33 +95,29 @@ definition {
...
+				ACSettings.deleteAllProperties();
+			}
+
 			ACProperties.assertNoProperties();
 		}
 
-		task ("Create two new properties") {
-			for (var propertyName : list "${propertyNameList}") {
-				ACProperties.addNewProperty(propertyName = "${propertyName}");
+		task ("Asset that there are no more properties on the sites page") {
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:115]: Small typo change Asset to Assert. Besides that LGTM!

**Review by joshitagaki — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:115]: Thanks Josh!!!

**Review by manuferreira — APPROVED** [review]
LGTM! thanks Vini :smile:

**Comment by christianstokes** [comment]
Nice looks good.

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**